### DISCUSSION TITLE
Regarding hacking happening in web browser of personal computers

### DISCUSSION THREAD

**Discussion by Subbaraman-Sowmya** [community/community#17532] [Other Feature Feedback, Questions, & Ideas]
How people are not able to prevent the hacking occurring in personal computer web browsers even though repeated several times?

![image](https://user-images.githubusercontent.com/61080915/171098020-cd894fe8-b385-4a6d-9e01-34da8fa845cd.png)
[Hacking my web browser.docx](https://github.com/github/feedback/files/8802202/Hacking.my.web.browser.docx)

**Comment by Gilanal** [comment]
None yet

**Comment by QX11369** [comment]
beepboop pwd

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
